<launch>
  <include file="./roboteq_controllers.launch" />
  <node name="imu" pkg="razor_imu_9dof" type="node.py">
   <rosparam>
     device: /dev/imu
   </rosparam>
  </node>
  <include file="./sick_lms_200.launch" />
  <node name="base_odometry" pkg="base_control" type="odometry_publisher_node.py" />
  <node name="joystick" pkg="joy" type="joy_node" />
  <node name="joy_controller" pkg="base_control" type="joy_controller_node.py" />
  <node name="base_control" pkg="base_control" type="base_control_node.py" />
</launch>
